src/instruments/src/EFB/Presets/Widgets/LightPresets.tsx

```diff
@@ -82,11 +84,99 @@ export const LightPresets = () => {
                         />
                     ))}
                 </div>
+                <AutoLoadConfiguration namesMap={namesMap} storedNames={storedNames} />
             </ScrollableContainer>
         </div>
     );
 };
 
+type AutoLoadConfigurationProps = {
+    namesMap: Map<number, string>,
+    storedNames: string
+}
+
+const AutoLoadConfiguration = (props: AutoLoadConfigurationProps) => {
+    const { showModal } = useModals();
+
+    const [autoLoadPreset, setAutoLoadPreset] = usePersistentNumberProperty('LIGHT_PRESET_AUTOLOAD', 0);
+
+    // State for persistent copy of autoload preset IDs
+    const [autoLoadDayPresetID, setAutoLoadDayPresetID] = usePersistentNumberProperty('LIGHT_PRESET_AUTOLOAD_DAY', 0);
+    const [autoLoadDawnDuskPresetID, setAutoLoadDawnDuskPresetID] = usePersistentNumberProperty('LIGHT_PRESET_AUTOLOAD_DAWNDUSK', 0);
+    const [autoLoadNightPresetID, setAutoLoadNightPresetID] = usePersistentNumberProperty('LIGHT_PRESET_AUTOLOAD_NIGHT', 0);
+
+    const [presetSelectionOptions, setPresetSelectionOptions] = useState([{ value: 0, displayValue: t('Presets.InteriorLighting.AutoLoadNoneSelection') }]);
+
+    // Creates the option list for the selections, ignoring any that are the default NoName title
+    const generatePresetSelectionOptions = () => {
+        const options: Array<{value: number, displayValue: string}> = [{ value: 0, displayValue: t('Presets.InteriorLighting.AutoLoadNoneSelection') }];
+        props.namesMap.forEach((value, key) => {
+            options.push({ value: key, displayValue: value });
+        });
+        return options;
+    };
+
+    const showAutoLoadHelp = () => {
```

Comment:
Not sure we need this - we have docs for this. I was thinking about removing it from Lighting Presets as well. It is the only place in the EFB we have done this. 

Especially I do not think such long texts should be translated.

I propose to remove this but it is up to you.